ucm2/USB-Audio/MOTU/M4-HiFi-Stereo-Mono.conf

```diff
@@ -0,0 +1,139 @@
+Include.pcm_split.File "/common/pcm/split.conf"
+
+Macro [
+	{
+		SplitPCM {
+			Name "m4_stereo_out"
+			Direction Playback
+			Channels 2
+			HWChannels 4
+			HWChannelPos0 FL
+			HWChannelPos1 FR
+			HWChannelPos2 FC
+			HWChannelPos3 LFE
+		}
+	}
+	{
+		SplitPCM {
+			Name "m4_mono_in"
+			Direction Capture
+			Channels 1
+			HWChannels 4
+			HWChannelPos0 FL
+			HWChannelPos1 FR
+			HWChannelPos2 FC
+			HWChannelPos3 LFE
+		}
+	}
+]
+
+SectionDevice."Line1-2" {
```

Comment:
Please, use only names from: https://github.com/alsa-project/alsa-lib/blob/master/include/use-case.h#L112-L144

So, `Line1` `Line2` ... and `Mic1` `Mic2` ... The description in Comment is for users.

Reply:
Thanks for the tip. I tested the profile and it's working properly so I updated the pull request accordingly. I think this profile is good enough since it represents the actual hardware IO.

Extra question for further enhancing this profile. I wanted to add the stereo inputs so I tried two different approaches. First I tried to have two SectionUseCase."HiFi" as below. But only one (the stereo-stereo one) shows up in the pipewire.
```
Comment "MOTU M4"

SectionUseCase."HiFi" {
	Comment "Analog Stereo Outputs + Mono Inputs"
	File "/USB-Audio/MOTU/M4-HiFi-Stereo-Mono.conf"
}
SectionUseCase."HiFi" {
	Comment "Analog Stereo Outputs + Stereo Inputs"
	File "/USB-Audio/MOTU/M4-HiFi-Stereo-Stereo.conf"
}

Define.DirectPlaybackChannels 4
Define.DirectCaptureChannels 4

Include.dhw.File "/common/direct.conf"
```

So I tried to add the stereo inputs alongside the mono ones in the default profile like below. But by doing so, the mono inputs won't capture sound anymore. The new stereo input works correctly though. 
```
Macro [
	...
	{
		SplitPCM {
			Name "m4_stereo_in"
			Direction Capture
			Channels 2
			HWChannels 4
			HWChannelPos0 FL
			HWChannelPos1 FR
			HWChannelPos2 FC
			HWChannelPos3 LFE
		}
	}
]

SectionDevice."In1L1R" {
	Comment "Mic In 1L + 1R"

	Value {
		CapturePriority 100
		CaptureMixer "default:${CardId}"
		CaptureMixerElem "Input 1"
	}
	Macro.pcm_split.SplitPCMDevice {
		Name "m4_stereo_in"
		Direction Capture
		HWChannels 4
		Channels 2
		Channel0 0
		Channel1 1
		ChannelPos0 FR
		ChannelPos0 FL
	}
}
```

Do you have any suggestions or examples on how to do this? It is not critical but it's a nice option to have for those who don't want to mess with patch bays every time.

Reply:
It should work. You can test the alsa devices directly using `arecord` or any other native ALSA application - dump the library config - `alsaucm -c hw:M4 getval _alibcfg > a.conf` - copy only pcm blocks to `~/.asoundrc` and use the `CapturePCM` device like `alsaucm -c hw:M4 getval CapturePCM/Mic1/HiFi` without `__ucm0001.` prefix.

Reply:
> First I tried to have two SectionUseCase."HiFi" as below.

You cannot specify two verbs with the identical identifier. Use "HiFi1" and "HiFi2" or so.

Reply:
I added the stereo inputs and tested them. It's working correctly.